sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -237,6 +237,7 @@ export default defineComponent({
               break;
           }
         } else {
+          showNav.value = false;
```

Comment:
I don't think we need this line as it was already defined as false on line 224.

Reply:
updated, I'd read the else as incorrectly a level out earlier